src/basic/parse-util.h

```diff
@@ -11,6 +11,26 @@
 
 typedef unsigned long loadavg_t;
 
+typedef enum Pcr {
+        PCR_CORE_FM,
+        PCR_PLAT_CONFIG,
+        PCR_EXT_CODE,
+        PCR_EXT_FM,
+        PCR_BOOT_LD,
+        PCR_GPT_PT,
+        PCR_POWER_ST,
+        PCR_SECURE_BT,
+        PCR_LINUX_KN = 9,
+        PCR_IMA_ST,
+        PCR_SYSTEMD_KN,
+        PCR_SYSTEMD_BT,
+        PCR_SYSTEMD_STB,
+        PCR_SHIM_CT,
+        PCR_LUKS_KEY,
```

Comment:
> this should not use automatic number assignment, since the numbers have meaning here. please specify the literal value for each entry.

Thank you sir. On it.

Reply:
> number 15 contains substantially more than the LUKS key...

Sorry to bother sir? but could you suggest a name for PCR 15? volumekey?
Thank you for taking the time.

I love reviews from this community.